lib/pure/strscans.nim

```diff
@@ -129,7 +129,7 @@ to use prefix instead of postfix operators.
 ``+E``           One or more
 ``?E``           Zero or One
 ``E{n,m}``       From ``n`` up to ``m`` times ``E``
-``~Ε``           Not predicate
+``~E``           Not predicate
```

Comment:
Some Unicode E introduced here? Or removed?

Reply:
Removed. That was Greek letter.